python: can't open file '../../parallel_prepocessing.py': [Errno 2] No such file or directory
sed: can't read varfile.txt: No such file or directory
sed: -e expression #1, char 3: invalid usage of line address 0
usage: parallel_single_run.py [-h] --varnames VARNAMES VARNAMES
parallel_single_run.py: error: argument --varnames: expected 2 arguments
